bin/build-project.ps1

```diff
@@ -34,22 +35,23 @@ if (-not (Test-Path $msbuild)) {
     throw "msbuild not found, cannot compile library! Check your .NET installation health, then try again. Path checked: $msbuild"
 }
 
-$msbuildOptions = ""
 if ($env:APPVEYOR -eq 'True') {
-    $msbuildOptions = '/logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"'
+    $MsbuildOptions = $MsbuildOptions + '/logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll" '
     $msbuildConfiguration = 'Debug'
 
     if (-not (Test-Path "C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll")) {
         throw "msbuild logger not found, cannot compile library! Check your .NET installation health, then try again. Path checked: $msbuild"
     }
 }
 
+$MsbuildOptions = $MsbuildOptions + '/logger:BinaryLogger,"{0}";"{1}" ' -f (Join-Path $PSScriptRoot 'StructuredLogger.dll'), (Resolve-Path '..\msbuild.bin.log').Path  
+
 if ( -not (Test-Path $ProjectPath)) {
     throw new-object 'System.IO.FileNotFoundException' 'Could not file project or solution', $ProjectPath
 }
 
-Write-Verbose -Message "Building the library with command $msbuild $ProjectPath /p:Configuration=$msbuildConfiguration $msbuildOptions /t:$MsBuildTarget"
-& $msbuild $ProjectPath "/p:Configuration=$msbuildConfiguration" $msbuildOptions "/t:$MsBuildTarget"
+Write-Verbose -Message "Building the library with command $msbuild $ProjectPath /p:Configuration=$msbuildConfiguration $MsbuildOptions /t:$MsBuildTarget"
+& $msbuild $ProjectPath "/p:Configuration=$msbuildConfiguration" $MsbuildOptions "/t:$MsBuildTarget"
```

Comment:
You aren't capturing msbuild errors, if the build fails, you will not be sending MSBLOC your failed build log.

https://github.com/justaprogrammer/MSBuildLogOctokitChecker/blob/master/docs/integrating.md#integrating-on-appveyor

Reply:
This is a good point, I think I need to use `on_failure:` to artefacts of a failed build.

Reply:
@zippy1981 are you going to make this change?